static/js/components/CourseList.js

```diff
@@ -1,22 +1,30 @@
 import React from 'react';
 import _ from 'lodash';
-
+import Button from 'react-bootstrap/lib/Button';
```

Comment:
we usually keep a blank line between dependency imports (react and so on) and our own code (the `util.js` import below this)